docs/get_started/install.md

```diff
@@ -424,13 +443,214 @@ $ sudo python setup.py install
 </div>
 <!-- END - Linux Python GPU Installation Instructions -->
 
-<div class="linux macos">
+<!-- START - MacOS Python CPU Installation Instructions -->
 
-# Validate MXNet Installation
+<div class="macos">
+  <div class="python">
+    <div class="cpu">
+
+The following installation instructions have been tested on OSX Sierra and El Capitan.
+
+
+**Prerequisites**
+
+If not already installed, [download and install Xcode](https://developer.apple.com/xcode/) for macOS. [Xcode](https://en.wikipedia.org/wiki/Xcode) is an integrated development environment for macOS containing a suite of software development tools like C/C++ compilers, BLAS library and more.
+
+<div class="virtualenv">
+<br/>
+
+**Step 1**  Install prerequisites - Homebrew, python development tools.
+
+```bash
+# Install Homebrew
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+$ export PATH=/usr/local/bin:/usr/local/sbin:$PATH
```

Comment:
Yes. This is a transient issue with PATH + homebrew installed packages. Setting these paths at the start of PATH, will help to make it more deterministic and fool proof.